### ISSUE TITLE
Update package prettify

### ISSUE THREAD

**Issue by ChienDevIT** [cdnjs/cdnjs#13082]
Today I accidentally used and discovered that the prettify package at CDNJS is outdated.
I know you are very busy so I helped you a little. However, the cdnjs package is too large so I can not clone to my machine. I created another package, you just clone and merge it.

https://github.com/ChienDevIT/code-prettify

Thanks,

**Comment by sashberd** [comment]
@ChienDevIT  if You are talking about this [lib](https://github.com/google/code-prettify). I do not know if we ca host it. 
@PeterDaveHello this lib has tags as date, moreover, there is no git autoupdate in package,json of lib. Files that should be hosted are ion loader folder.
Can we create autoupdate config with tags as dates?

**Comment by ChienDevIT** [comment]
Yes, I forgot to create a tag when done. Really sorry you. But I do not know how to create a card so it's easy to understand. So I hope you help me this part. Thank you.

**Comment by sashberd** [comment]
@ChienDevIT We cannot host you fork. We have to check if we can host origin lib. Lets wait fo @PeterDaveHello answer
